src/interfaces/node.cpp

```diff
@@ -228,6 +229,32 @@ class NodeImpl : public Node
         return wallets;
 #else
         throw std::logic_error("Node::getWallets() called in non-wallet build.");
+#endif
+    }
+    std::pair<std::unique_ptr<Wallet>, std::string> loadWallet(const std::string& wallet_file) override
+    {
+#ifdef ENABLE_WALLET
+        try {
+            std::string dummy_warning;
```

Comment:
This seems almost exactly duplicated from `loadwallet()` in rpcwallet.cpp. Does it make sense to factor it out to avoid repetition?

Reply:
@jnewbery done in #9888dc2.